<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Players</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="lePlayer"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lePlayerG">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="inputMask">
        <string>000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lPlayer">
       <property name="text">
        <string>Player Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lPlayerG">
       <property name="text">
        <string>Goals</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnClean">
     <property name="text">
      <string>Clean</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lePlayer</tabstop>
  <tabstop>lePlayerG</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnClean</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnSearch</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>Dialog2</receiver>
   <slot>callFunctionPlayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClean</sender>
   <signal>clicked()</signal>
   <receiver>Dialog2</receiver>
   <slot>clearAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSearch</sender>
   <signal>clicked()</signal>
   <receiver>Dialog2</receiver>
   <slot>searchPlayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchPlayer()</slot>
  <slot>callFunctionPlayer()</slot>
  <slot>clearAll()</slot>
 </slots>
</ui>
